src/App.tsx

```diff
@@ -1,39 +1,62 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type Props = {};
 
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
+type State = {
+  hasClock: boolean;
+  clockName: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state: State = {
```

Comment:
Maybe, if you don't use props in this component you can just put an empty object instead (<{}, State>), so you won't need special type for them as well.